### PR TITLE
update database test data

### PR DESCRIPTION
**Pull request by mafh314**
Fixes [#181](https://github.com/kubearchive/kubearchive/issues/181)

### CODE REVIEW

**Review by maruiz93 — CHANGES_REQUESTED** [review]
If we are inserting uuid from the resource, shouldn't we specify in the table DDL, that the field shouldn't be empty?

```
CREATE TABLE IF NOT EXISTS public.resource (
-               "uuid" uuid PRIMARY KEY,
+               "uuid" uuid PRIMARY KEY NOT NULL,
                "api_version" varchar NOT NULL,
```

EDIT: You are right, based on the comments you wrote, this change isn't needed at all, I'll approve the PR

File: database/resource.sql [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
\ No newline at end of file
+INSERT INTO public.resource (api_version,"cluster",cluster_uid,kind,"name","namespace",resource_version,created_ts,updated_ts,cluster_deleted_ts,"data","uuid") VALUES
```
[the remark this answers is not in the document]
Comment (rh-hemartin) [reply database/resource.sql:1]: Those changes are more extensive than just removing them here, I would like that we address that in another place for an easier review.

**Comment by ggallen** [comment]
> If we are inserting uuid from the resource, shouldn't we specify in the table DDL, that the field shouldn't be empty?

By definition a primary key can not be NULL.  I think `PRIMARY KEY NOT NULL` might be invalid SQL.

**Comment by rh-hemartin** [comment]
> > If we are inserting uuid from the resource, shouldn't we specify in the table DDL, that the field shouldn't be empty?
> 
> By definition a primary key can not be NULL. I think `PRIMARY KEY NOT NULL` might be invalid SQL.

I think its valid because it is on `main` and its currently working.

**Comment by ggallen** [comment]
> > > If we are inserting uuid from the resource, shouldn't we specify in the table DDL, that the field shouldn't be empty?
> > 
> > 
> > By definition a primary key can not be NULL. I think `PRIMARY KEY NOT NULL` might be invalid SQL.
> 
> I think its valid because it is on `main` and its currently working.

Then why is there any change proposed here?

**Comment by rh-hemartin** [comment]
> Then why is there any change proposed here?

Its not being proposed.  @maruiz93 added a code block that resembles a diff to ask for that change.

**Review by beerparty — APPROVED** [review]

**Review by maruiz93 — APPROVED** [review]

**Comment by mafh314** [comment]
> Edit: I think now I understand what you meant. You are both right (by definition a primary key can not be NULL) and wrong (because its a valid SQL). **We are fixing the sample resources to have a uuid because it can't be null.**
This PR is only to fix the issue that we have with the test data. I didn't apply any change to the _init_db_ script. If any change, related to the Db schema, is needed,  we can a open a new issue.

**Review by rh-hemartin — APPROVED** [review]

**Pull request merged**